datastore/setup.py

```diff
@@ -53,8 +53,10 @@
 REQUIREMENTS = [
     'google-cloud-core >= 0.28.0, < 0.29dev',
     'google-api-core >= 0.1.1, < 0.2.0dev',
-    'google-gax >= 0.15.7, < 0.16dev',
-    'gapic-google-cloud-datastore-v1 >= 0.15.0, < 0.16dev',
+    'google-auth >= 1.0.2, < 2.0dev',
+    'google-gax >= 0.15.15, < 0.16dev',
+    'googleapis-common-protos[grpc] >= 1.5.2, < 2.0dev',
+    'requests >= 2.18.4, < 3.0dev',
```

Comment:
@lukesneeringer Why do we "repeat" ourselves here when all of these are in the deps for `google-api-core` / `google-cloud-core`?